### DISCUSSION TITLE
Resuming to watch file

### DISCUSSION THREAD

**Discussion by summerance** [vectordotdev/vector#13244] [Q&A] [answered]
<img width="1575" alt="截屏2022-06-17 16 37 14" src="https://user-images.githubusercontent.com/32301268/174708678-9eb4ec72-5beb-4984-a3ba-09a70fa6f125.png">
how to fix the problem that resuming to watch the file.   
I hope he can scan every time regardless of whether the file is repeated or not. Which parameter can I set and how?

**Comment by StephenWakely** [comment, marked as answer]
You can try setting the [fingerprint strategy](https://vector.dev/docs/reference/configuration/sources/file/#fingerprint.strategy) to be `device_and_inode`.
